test/plugins-service.ts

```diff
@@ -564,6 +576,8 @@ describe("plugins-service", () => {
 
 			options.debug = true;
 			options.release = false;
+			console.log("installed plugins in debug config!!! ");
```

Comment:
Are these left-over debug code or they are supposed to be here? In later case, consider using $logger